src/components/policy/policy_external/src/policy_table/enums.cc

```diff
@@ -888,6 +888,170 @@ bool EnumFromJsonString(const std::string& literal,
   return false;
 }
 
+const char* EnumToJsonString(FunctionID val) {
```

Comment:
I see that this method existed in regular policies, but not external? Was it used by regular policies before this feature at all?

Reply:
not sure about whether it was used before. this method was generated dynamically in code generation part of this feature. since dynamic generation was discarded, this method needed to be added for external mode as well.

Reply:
@JackLivio such methods exists for each policy enum. 
Previously EnumToJsonString was not used for FunctionID, and it was only declared, but not implemented. Now it needs to be implemented because of usage in  `RPCProtectionManagerImpl::IsExceptionRPC`. 
Now, this function implementation exists both in external and regular : 
https://github.com/smartdevicelink/sdl_core/blob/feature/rpc_message_protection/src/components/policy/policy_regular/src/policy_table/enums.cc#L915
https://github.com/smartdevicelink/sdl_core/blob/feature/rpc_message_protection/src/components/policy/policy_external/src/policy_table/enums.cc#L891